### DISCUSSION TITLE
How to list only posts within the selected category?

### DISCUSSION THREAD

**Discussion by nibl** [forever-jekyll/forever-jekyll#3] [Q&A] [answered]
Does anyone know how to change the category page so it only lists posts within the selected category? For example, /categories/#music would only list posts tagged music. I think this means passing the tag name as a parameter to the category page. But I'm not sure how to pass that parameter. It is presumably different than the current way of passing a link to a bookmark (e.g. #music).

**Comment by nibl** [comment, marked as answer]
I figured out that this question possibly  contradicts the nature of a *static* site generator. The  way to achieve this is to create a page (i.e. music.html) and use liquid to filter posts within the music category. Then use links to that page.

**Reply by forever-jekyll** [reply]
Hi there @nibl 
Really sorry for the belated reply.
Yes, you are right. It can be done but it varies from the user case to user case and there is hardly any one-size-fits-all formula that I know of that can achieve this.
You can share your custom code here if you wish. It'll be really helpful to many others.
